openslides/agenda/static/js/agenda/base.js

```diff
@@ -0,0 +1,122 @@
+(function () {
```

Comment:
We do not always just call an angular function. Sometimes there are extra functions defined in a file. I would say using a closure is always best practice to protect the global namespace.